### PR TITLE
Rachel Pavilanis - completed homework 

### PR DESCRIPTION
**Pull request by rpavilanis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +26,7 @@ end
 ```
 
 Snippet 2 - Big O:
+ANSWER Big O(2n)
```
Comment (sudocrystal) [inline README.md:29]: Drop coefficients --> O(n)

File: README.md [hunk 2]
```diff
@@ -57,6 +61,7 @@ def duplicates?(array)
 end
 ```
 
+ANSWER Big O(n**2)
```
Comment (sudocrystal) [inline README.md:64]: This one is actually O(n \* m) ... why?

File: README.md [hunk 3]
```diff
@@ -78,6 +84,7 @@ def print_array(array)
 end
 ```
 
+ANSWER Big O(n**2) (or maybe Big O(log n)?)
```
Comment (sudocrystal) [inline README.md:87]: O(n^2) is correct. What part of the loop made you think it might be log n?

File: README.md (hunk 2 above)
Comment (rpavilanis) [reply README.md:64]: Ahh...because the two loops don't depend on each other...?

**Pull request closed without merging**